www/nginx/Makefile

```diff
@@ -1,5 +1,5 @@
 PLUGIN_NAME=		nginx
-PLUGIN_VERSION=		1.34
+PLUGIN_VERSION=		1.35
```

Comment:
1.34 has not been released, better to batch with this one instead of opening another version.